README.md

```diff
@@ -103,6 +103,10 @@ and use to get a basic plugin experience:
 - [surround](include/clap/ext/draft/surround.h), inspect surround channel mapping
 - [ambisonic](include/clap/ext/draft/ambisonic.h), inspect ambisonic channel mapping
 
+## Third-party extensions
+
+- [cockos.reaper_extension](https://github.com/justinfrankel/reaper-sdk/blob/main/reaper-plugins/reaper_plugin.h#L138), access the [REAPER](http://reaper.fm) API
```

Comment:
If this list contains full extension IDs, then it may be a good precedent to wrap that ID in backticks. Might make it a bit easier to skim through the list once more third party extensions are added.

```suggestion
- [`cockos.reaper_extension`](https://github.com/justinfrankel/reaper-sdk/blob/main/reaper-plugins/reaper_plugin.h#L138), access the [REAPER](http://reaper.fm) API
```